services/getstream/getUnfilteredActivities.js

```diff
@@ -18,8 +18,6 @@ const feedSwitch = async (feed) => {
     case 'main_feed_following':
       return 'main_feed_f2';
     case 'main_feed_f2':
-      return 'main_feed_broad';
-    case 'main_feed_broad':
       return 'main_feed';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change in the return value of the 'main_feed_f2' case from 'main_feed_broad' to 'main_feed' might alter the behavior of your application if there are any specific functionalities associated with 'main_feed_broad'. Please ensure that this change aligns with your intended functionality.
```diff
-      return 'main_feed_broad';
+      return 'main_feed';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->